### DISCUSSION TITLE
!osu lazer Game Crashing

### DISCUSSION THREAD

**Discussion by 4219XX** [ppy/osu#15922] [Q&A] [answered]
Some beatmaps keep crashing the osu lazer client upon selecting them in the beatmap selection screen. It crashes in both single and multiplayer, I have deleted the maps and redownload them and the same thing happens. It happens on a lot of maps so it must be something major in common between them that causes the crash as there are many many more that cause the crash.
List Of Some Maps which cause this issue:  https://osu.ppy.sh/beatmapsets/1055990#osu/2303351   https://osu.ppy.sh/beatmapsets/956590#osu/2002605   https://osu.ppy.sh/beatmapsets/874575#osu/1849487

[database.log](https://github.com/ppy/osu/files/7652456/database.log)
[network.log](https://github.com/ppy/osu/files/7652457/network.log)
[performance.log](https://github.com/ppy/osu/files/7652458/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7652459/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7652460/updater.log)

**Comment by Ekital** [comment]
Can confirm this is a pure hard crash, no logs seem to be generated when it happens when I helped test it.

**Comment by frenzibyte** [comment]
This may be a video-related issue as all the beatmaps you linked have videos, can you try disabling `Use hardware acceleration` in video settings and see if the issue still persists?
```
2021-12-03 21:36:02 [verbose]: Couldn't create hardware video decoder context AV_HWDEVICE_TYPE_CUDA for codec h264_cuvid: Unknown error occurred (-1313558101)
2021-12-03 21:36:02 [verbose]: Couldn't create hardware video decoder context AV_HWDEVICE_TYPE_CUDA for codec h264: Unknown error occurred (-1313558101)
```

**Comment by Ekital** [comment]
Seems to have fixed it @frenzibyte .


Thanks.

**Comment by frenzibyte** [comment, marked as answer]
Good to hear, hardware acceleration should implicitly get disabled when failing to initialize it, that's tracked in https://github.com/ppy/osu-framework/issues/4857.
